lib/active_admin/resource/routes.rb

```diff
@@ -51,13 +55,24 @@ def instance_path(instance)
           routes.public_send route_name, *route_instance_params(instance)
         end
 
+        # @return [String] the path to the edit page of this resource
+        # @param instance [ActiveRecord::Base] the instance we want the path of
+        # @example "/admin/posts/1/edit"
+        def edit_instance_path(instance)
+          route_name = route_name(resource.resources_configuration[:self][:route_instance_name], action: :edit)
+
+          routes.public_send route_name, *route_instance_params(instance)
+        end
+
         private
 
         attr_reader :resource
 
-        def route_name(resource_path_name, suffix = 'path')
+        def route_name(resource_path_name, options = {})
+          suffix = options[:suffix] || 'path'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.